corehq/apps/app_manager/models.py

```diff
@@ -478,15 +478,17 @@ class AutoSelectCase(DocumentSchema):
 
 class LoadCaseFromFixture(DocumentSchema):
     """
-    fixture_nodeset:    FixtureDataType.tag
-    fixture_tag:        name of the column to display in the list
-    fixture_variable:   boolean if display_column actually contains the key for the localized string
-    case_property:      name of the column whose value should be saved when the user selects an item
-    arbitrary_datum_*:  adds an arbitrary datum with function before the action
+    fixture_nodeset:     nodeset that returns the fixture options to display
+    fixture_tag:         id of session datum where the result of user selection will be stored
+    fixture_variable:    value from the fixture to store from the selection
+    auto_select_fixture: boolean to autoselect the value if the nodeset only returns 1 result
+    case_property:       case property to filter on
+    arbitrary_datum_*:   adds an arbitrary datum with function before the action
```

Comment:
Don't know what I was doing when I originally documented these. Guessing I just copy and pasted them from some other documentation without thought